### DISCUSSION TITLE
Incorrect response with Local AI and k8sgpt

### DISCUSSION THREAD

**Discussion by dilipk8** [mudler/LocalAI#452] [Q&A]
**LocalAI version:**
quay.io/go-skynet/local-ai:latest

**Environment, CPU architecture, OS, and Version:**
5.13.0-1022-aws #24~20.04.1-Ubuntu SMP   x86_64 x86_64 x86_64 GNU/Linux
Kubernetes version - 1.22.4 

**Describe the bug**
I am trying to use k8sgpt with Local AI. My K8s cluster is a kops cluster on AWS. I configured local AI as per the docs using the ggml-gpt4all-j model.
I create an nginx pod with an invalid image and this was the k8sgpt analyze result, which is incorrect.

- Error: Back-off pulling image "nginedd"
The prompt is asking to simulate a Kubernet error message in English, delimited by triple dashes. The user is then asked to provide a step-by-step solution, in no more than 280 characters. The output should be in the format: Error (explain error here) Solution (step-by-step solution here).

**To Reproduce**
git clone https://github.com/go-skynet/LocalAI
cd LocalAI
wget https://gpt4all.io/models/ggml-gpt4all-j.bin -O models/ggml-gpt4all-j
cp -rf prompt-templates/ggml-gpt4all-j.tmpl models/
docker-compose up -d --pull always

Next - Use a K8s cluster, install and configure k8sgpt cli with local AI and run k8sgpt analyze and see the result.

**Expected behavior**
Valid message which tell us how to troubleshoot the invalid image error.

**Comment by mudler** [comment]
seems more a limitation on the model. try for Koala for instance https://github.com/go-skynet/model-gallery#gpt-koala

**Reply by dilipk8** [reply]
@mudler , okay,but just wondering it looks like the same model worked fine, in your video here - https://www.youtube.com/watch?v=PKrDNuJ_dfE
I will try another model but is there something am missing out or something that i can double-check

**Comment by vitorstone** [comment]
So you installed LocalAI on a container outside of the cluster, and then configured the k8sgpt operator on the cluster. How did you configure it so that it works with the LocalAI API outside the cluster?
